admin/installer/cloudformation.py

```diff
@@ -70,19 +70,29 @@ def _sibling_lines(filename):
 # Keys corresponding to CloudFormation user Inputs.
 access_key_id_param = template.add_parameter(Parameter(
     "AmazonAccessKeyID",
-    Description="Your Amazon AWS access key ID",
+    Description="Your Amazon AWS access key ID (mandatory)",
```

Comment:
This is better, but I can still leave the field empty and click next.
It seems unbelievable that you can't just flag a Parameter as required....but the documentation suggests it's not possible.
Perhaps we can create a "Condition" which checks that the key and secret key are not empty before continuing with the stack creation?
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/conditions-section-structure.html